docs/wip/exchange-onboarding.md

```diff
@@ -0,0 +1,352 @@
+
+## Sending Money
+
+In order to send money from one of your account to another address, you can create a new payment transaction using the POST /api/v1/transactions endpoint as follow:
+
+    curl -X POST https://localhost:8090/api/v1/transactions \
+         -H "Content-Type: application/json; charset=utf-8" \
+         -H "Accept: application/json; charset=utf-8"       \
+         --cacert ./scripts/tls-files/ca.crt                \
+         -d '{                                              \
+      "destinations": [{                                    \
+        "amount": 14,                                       \
+        "address": "A7k5bz1QR2...Tx561NNmfF"                \
+      }],                                                   \
+      "source": {                                           \
+        "accountIndex": 0,                                  \
+        "walletId": "Ae2tdPwUPE...8V3AVTnqGZ"               \
+      }                                                     \
+    }'
+
+Note that, in order to perform a transaction, you need to have some existing coins on the source account! Beside, the Cardano API is designed to accomodate multiple recipients payments out-of-the-box; notice how destinations is a list of addresses.
+
+When the transaction succeeds, funds are becomes unavailable from the sources addresses, and available to the destinations in a short delay. Note that, you can at any time see the status of your wallets by using the GET /api/v1/transactions/{{walletId}} endpoint as follow:
+
+    curl -X GET https://localhost:8090/api/v1/wallets/{{walletId}}?account_index=0  \
+         -H "Accept: application/json; charset=utf-8"                               \
+         --cacert ./scripts/tls-files/ca.crt                                        \
+
+We have here constrainted the request to a specific account, with our previous transaction the output should look roughly similar to this:
+
+    {
+        "status": "success",
+        "data": [
+            {
+                "amount": 14,
+                "inputs": [{
+                  "amount": 14,
+                  "address": "DdzFFzCqrh...fXSru1pdFE"
+                }],
+                "direction": "outgoing",
+                "outputs": [{
+                  "amount": 14,
+                  "address": "A7k5bz1QR2...Tx561NNmfF"
+                }],
+                "confirmations": 42,
+                "id": "43zkUzCVi7...TT31uDfEF7",
+                "type": "local"
+            }
+        ],
+        "meta": {
+            "pagination": {
+                "totalPages": 1,
+                "page": 1,
+                "perPage": 10,
+                "totalEntries": 1
+            }
+        }
+    }
+
+## Where can I find the API documentation?
+
+Run the latest wallet and go to <https://127.0.0.1:8090/docs/v1/index>.
```

Comment:
Are the V0 docs no longer available anywhere? Until all exchanges are migrated to V1 we need to keep them available, while simultaneously discouraging their use for new exchanges.

Reply:
Mention the `walletListen` configuration option in `./custom-wallet-config.nix`.